### PR TITLE
Video redirect class

### PR DESCRIPTION
**Pull request by collin**
@mparent61 This passes the existing test cases.

My thinking is to extend this class in onion and override:

```py
def choose_video_for_redirect(self, videos):
```

To filter on the feature_type. ( in python, not as a db query )

Thoughts?

### CODE REVIEW

**Comment by mparent61** [comment]
Yeah this is a good workaround. I would add plenty of comments that this will be removed once unique constraint is in place.

File: bulbs/videos/views.py [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+        videos = self.get_videos_for_redirect(videohub_ref_id)
+        video = self.choose_video_for_redirect(videos)
+
+        if video is None:
+            raise Http404("No video found with the provided videohub_ref id.")
+
+        return HttpResponseRedirect(video.get_absolute_url())
+
+    def choose_video_for_redirect(self, videos):
+        try:
```
Comment (mparent61) [inline bulbs/videos/views.py]:
```
if videos:
    return videos[0]
```

File: bulbs/videos/views.py [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+    def get(self, request, videohub_ref_id):
+        videos = self.get_videos_for_redirect(videohub_ref_id)
+        video = self.choose_video_for_redirect(videos)
+
+        if video is None:
+            raise Http404("No video found with the provided videohub_ref id.")
+
+        return HttpResponseRedirect(video.get_absolute_url())
+
+    def choose_video_for_redirect(self, videos):
```
Comment (mparent61) [inline bulbs/videos/views.py]: maybe rename to "temporary_workaround_to_choose_from_duplicate_videos()"

**Comment by collin** [comment]
@benghaziboy @MichaelButkovic Lookin' for that second 👍

**Comment by MichaelButkovic** [comment]
LGTM

**Pull request merged**